app/helpers/middleware_server_helper/textual_summary.rb

```diff
@@ -44,4 +44,20 @@ def textual_product
   def textual_version
     @record.properties['Version']
   end
+
+  def textual_lives_on
+    lives_on_ems = @record.try(:lives_on).try(:ext_management_system)
+    return nil if lives_on_ems.nil?
+    lives_on_entity_name = _("Virtual Machine")
+     {
+         :label      => "Underlying #{lives_on_entity_name}",
+         :image      => "vendor-#{lives_on_ems.image_name}",
+         :value      => "#{@record.lives_on.name}",
```

Comment:
I'll check. from a first glance looks like no reason. but perhaps sometimes this field contains something non stringy